build/Tests/UnitTests/Starboard/Starboard.UnitTests.vcxproj

```diff
@@ -226,6 +226,7 @@
     <ClCompile Include="$(StarboardBasePath)\tests\unittests\EntryPoint.cpp" />
   </ItemGroup>
   <ItemGroup>
+    <ClangCompile Include="..\..\..\..\Tests\UnitTests\Starboard\AutoCFTests.cpp" />
```

Comment:
I'll try to fix the casing on this directory name. Case-sensitive filenames are very much not happy about this.